cmd/sonobuoy/app/imageversion.go

```diff
@@ -77,19 +78,25 @@ func (c *ConformanceImageVersion) Get(client discovery.ServerVersionInterface) (
 			return "", errors.Wrap(err, "couldn't retrieve server version")
 		}
 
-		if err := validateVersion(version.GitVersion); err != nil {
+		parsedVersion, err := validateVersion(version.GitVersion)
+		if err != nil {
 			return "", err
 		}
 
+		segments := parsedVersion.Segments()
```

Comment:
@liztio could you put a comment here to follow up on why GKE diverged on minor version.  It's super odd.